.changeset/good-eagles-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"geohub": major
+---
+
+Simulate Sliders
```

Comment:
Could you merge two changesets file into a changeset file?

also change major to minor. This PR cannot be a major release of geohub. it is just a new feature.

change changelog text like `feat: added new CEEI dashboard`